spec.html

```diff
@@ -26748,19 +26748,20 @@ <h1>Date Time String Format</h1>
 THH:mm:ss.sss
         </pre>
         <p>All numbers must be base 10. If the `MM` or `DD` fields are absent `"01"` is used as the value. If the `HH`, `mm`, or `ss` fields are absent `"00"` is used as the value and the value of an absent `sss` field is `"000"`. When the time zone offset is absent, date-only forms are interpreted as a UTC time and date-time forms are interpreted as a local time.</p>
-        <p>Illegal values (out-of-bounds as well as syntax errors) in a format string means that the format string is not a valid instance of this format.</p>
+        <p>Illegal values (out-of-bounds as well as syntax errors) in a format string mean that the format string is not a valid instance of this format.</p>
         <emu-note>
-          <p>As every day both starts and ends with midnight, the two notations `00:00` and `24:00` are available to distinguish the two midnights that can be associated with one date. This means that the following two notations refer to exactly the same point in time: `1995-02-04T24:00` and `1995-02-05T00:00`</p>
+          <p>As every day both starts and ends with midnight, the two notations `00:00` and `24:00` are available to distinguish the two midnights that can be associated with one date. This means that the following two notations refer to exactly the same point in time: `1995-02-04T24:00` and `1995-02-05T00:00`. ISO 8601 allows an hour of 24 only to indicate the end of a calendar day within a time interval, but ECMAScript permits it in representations of single time points.</p>
         </emu-note>
         <emu-note>
-          <p>There exists no international standard that specifies abbreviations for civil time zones like CET, EST, etc. and sometimes the same abbreviation is even used for two very different time zones. For this reason, ISO 8601 and this format specifies numeric representations of date and time.</p>
+          <p>There exists no international standard that specifies abbreviations for civil time zones like CET, EST, etc. and sometimes the same abbreviation is even used for two very different time zones. For this reason, both ISO 8601 and this format specify numeric representations of time zone offsets.</p>
         </emu-note>
 
-        <emu-clause id="sec-extended-years">
-          <h1>Extended Years</h1>
-          <p>ECMAScript requires the ability to specify 6 digit years (extended years); approximately 285,426 years, either forward or backward, from 01 January, 1970 UTC. To represent years before 0 or after 9999, ISO 8601 permits the expansion of the year representation, but only by prior agreement between the sender and the receiver. In the simplified ECMAScript format such an expanded year representation shall have 2 extra year digits and is always prefixed with a + or - sign. The year 0 is considered positive and hence prefixed with a + sign.</p>
+        <a id="sec-extended-years"></a>
```

Comment:
Is it valuable to preserve old anchors like this? And if so, am I doing it right?

Reply:
It's valuable to preserve them, but the preferred way would be to add `oldids="sec-extended-years"` to the following `<emu-clause>` tag. (The rendering process then takes care of generating an anchor for the old id.)